nvflare/app_common/workflows/cyclic_ctl.py

```diff
@@ -139,7 +146,9 @@ def control_flow(self, abort_signal: Signal, fl_ctx: FLContext):
                 )
                 self.persistor.save(self._last_learnable, fl_ctx)
                 self.log_debug(fl_ctx, "Ending current round={}.".format(self._current_round))
-                self._engine.persist_components(fl_ctx, completed=False)
+                if self._snapshot_every_n_rounds != 0 and self._current_round % self._snapshot_every_n_rounds == 0:
```

Comment:
To be consistent with SAG, should we add another arg for persist_every_n_rounds to control self.persistor.save frequency?

Reply:
thanks, fixed